Userland/Libraries/LibJS/JIT/Compiler.cpp

```diff
@@ -2722,7 +2727,19 @@ OwnPtr<NativeExecutable> Compiler::compile(Bytecode::Executable& bytecode_execut
         dbgln("\033[32;1mJIT compilation succeeded!\033[0m {}", bytecode_executable.name);
     }
 
-    auto executable = make<NativeExecutable>(executable_memory, compiler.m_output.size(), mapping);
+    // Build executable
```

Comment:
Comments should explain why code does something, not what it does (See CodingStyle.md).
You can just leave comments like this out, since it's obvious what `build_gdb_image` and `make<NativeExecutable>` do.